lms/envs/common.py

```diff
@@ -3175,6 +3175,7 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
     ('consent', None),
     ('integrated_channels.integrated_channel', None),
     ('integrated_channels.degreed', None),
+    ('integrated_channels.canvas', None),
```

Comment:
I didn't check the actual edx-enterprise PR proceeding this, but does this do anything at the moment? Like is this going to try to run canvas (and would that cause errors)?

Reply:
good question, that is indeed what I wanted to find out too. I do not think anything should happen based on what George had indicated in a slack thread. since there is no enterprise customer that will find a canvas channel registered to it, but I am also trying to find out more about how that jenkins task works and at least be able to track results of it once this merges

Reply:
Ahha! I just found this, I think Canvas will not yet be included in the list of channel used in the tasks since I enver added it to this

https://github.com/edx/edx-enterprise/blob/af5b13a109f611375c3fffbf97aa236cf0fd38c7/integrated_channels/integrated_channel/management/commands/__init__.py#L17

Maybe we should add this entry once we are done testing locally but not until then?

Reply:
Since in prod the table `canvas_canventerprisecustomerconfiguration` is not yet going to contain any entries, even if we added this, it would find no entries and no action would be taken for canvas, yet

This is good and it means that we need to add a follow up (enable canvas in prod) task to actually do this piece and enable canvas to be activated for a test customer in stage maybe for the first round of testing